src/modules/UI/components/ErrorAlert/actions.js

```diff
@@ -1,13 +1,25 @@
 // @flow
+import { Alert } from 'react-native'
+
+import * as actions from '../../../../actions/indexActions.js'
+import s from '../../../../locales/strings.js'
+import type { Dispatch } from '../../../../modules/ReduxTypes'
 
 const PREFIX = 'UI/components/ErrorAlert/'
 export const DISPLAY_ERROR_ALERT = PREFIX + 'DISPLAY_ERROR_ALERT'
 export const DISMISS_ERROR_ALERT = PREFIX + 'DISMISS_ERROR_ALERT'
 
-export const displayErrorAlert = (message: string) => ({
+export const displayErrorAlertStore = (message: string) => ({
```

Comment:
the action creator `displayErrorAlert` is called at least 12 times in the code. I needed to centralize the possibility of any number of those throwing an error 'Invalid OTP Token' and changing the error message to being something more explanatory and user friendly. However `src/libs/ErrorAlert.js` need the call to displayErrorAlert to be solely an action not a thunk. So in order to keep it from breaking I copied `displayErrorAlert` and renamed it `displayErrorAlertStore` since it was being called from a util using `store.dispatch`. Now the filtering is centralized and the utility is not broken either.